src/gflags.h.in

```diff
@@ -429,11 +429,57 @@ extern GFLAGS_DLL_DECL void ShutDownCommandLineFlags();
 // people can't DECLARE_int32 something that they DEFINE_bool'd
 // elsewhere.
 
+enum FlagValueType {
```

Comment:
That may be true, but in the header we only needed the declaration of the `FlagRegisterer` header. It is not uncommon to then provide only specific explicit template instantiations in the `.cc` file. And in our case, `FlagRegisterer` isn't really a public API, it is used by the `DEFINE_` macros and there is only a limited number of types for which we needed an explicit template instantiation in the `gflags.cc` file...